ubuntu-deploy/index.md

```diff
@@ -0,0 +1,414 @@
+      DB_CLIENT: 'sqlite3'
+      DB_FILENAME: '/directus/database/data.db'
+      WEBSOCKETS_ENABLED: 1
+      # Add your other settings
+```
+
+:::info Security Note
+
+Ensure you adjust the email and set a strong password for the admin user.
+
+:::
+
+Use `scp` (Secure Copy Protocol) to upload the local folder to your server.
+
+From your local machine terminal, run:
+
+```bash
+scp -r /path/to/your/local/directus/folder username@server_ip:/path/to/your/remote/folder
+```
+
+Replace _/path/to/your/local/directus/folder_ with the actual local path of your Directus application directory, and _/path/to/your/remote/folder_ with the respective path on your server.
+
+Also, replace `username` with your server's username and `server_ip` with your server's public IP address.
+
+In the example below, I am copying the whole Directus folder to the home directory(`~`) of my server.
+
+![Copying files to the server][image-6]
+
+:::info Database Note
+
+Note that the database used in this tutorial is SQLite. For other types of databases like MySQL and PostgreSQL, you might have to create a database dump and export the dump to your remote server.
+
+:::
+
+## Preparing Your Ubuntu Server
+
+Access your Ubuntu server from your local machine terminal via SSH:
+
+```bash
+ssh username@server_ip
+```
+
+In your server terminal, run the following commands:
+
+```bash
+sudo apt update
+sudo apt upgrade -y
+```
+
+This command updates and upgrades all existing packages to their latest versions.
+
+## Installing Docker and Docker Compose
+
+To install Docker, run the following commands:
+
+```bash
+sudo apt install docker.io -y
+sudo systemctl start docker
+sudo systemctl enable docker
+```
+
+This installs Docker and enables the Docker service.
+To check if Docker is running, run:
+
+```bash
+sudo systemctl status docker
+```
+
+To install Docker Compose, run the following commands:
+
+```bash
+sudo apt install docker-compose -y
+```
+
+To verify that the installation was successful, you can run:
+
+```bash
+docker-compose --version
+```
+
+## Start your Directus Application
+
+Run these commands to allow incoming traffic on port 8055:
+
+```bash
+sudo ufw allow 8055/tcp
+sudo ufw enable
+```
+
+Start your Directus application using Docker Compose:
+
+```bash
+cd /path/to/your/directus/folder
+sudo docker-compose up
+```
+
+![Running docker compose][image-11]
+
+On the initial run, Docker will fetch the necessary image from the registry before launching your Directus application.
+
+Your application should now be accessible at `http://your_server_ip:8055`.
+
+![Directus application accessed with the server ip at port 8055][image-2]
+
+:::info Error Debugging
+
+If you encounter any error e.g `SQLITE_CANTOPEN: unable to open database file`, it is probably due to permission issues. You can try running this command to fix it:
+
+```bash
+sudo chown -R $USER:$USER /path/to/your/directus/folder
+
+```
+
+More info about this error in this [issue][issue]
+
+:::
```

Comment:
````suggestion
:::info Debugging SQLITE_CANTOPEN

If you encounter any error e.g `SQLITE_CANTOPEN: unable to open database file`, it is probably due to [permission issues][issue]. You can run this command to fix it:

```bash
sudo chown -R $USER:$USER /path/to/your/directus/folder
```

:::
````

A bit more concise